weather.noaa/resources/language/resource.language.en_GB/strings.po

```diff
@@ -0,0 +1,502 @@
+# Kodi Media Center language file
```

Comment:
language/resource.language.en_gb/strings.po not language/resource.language.en_GB/strings.po  (consistency with other addons)